db/write_controller_test.cc

```diff
@@ -26,115 +27,206 @@ class WriteControllerTest : public testing::Test {
   std::shared_ptr<TimeSetClock> clock_;
 };
 
-TEST_F(WriteControllerTest, ChangeDelayRateTest) {
-  WriteController controller(40000000u);  // also set max delayed rate
-  controller.set_delayed_write_rate(10000000u);
+// Make tests easier to read
+#define MILLION *1000000u
+#define MB MILLION
+#define MBPS MILLION
+#define SECS MILLION  // in microseconds
+
+TEST_F(WriteControllerTest, BasicAPI) {
```

Comment:
Is/Should there be a test that:
- Validates that GetDelay returns 0 if there are stop tokens
- Validates that GetDelay returns 0 if there are no delay tokens
- Cancels a stop token and checks the delay